cmd/osv-scanner/main_test.go

```diff
@@ -948,7 +948,7 @@ func TestRun_SubCommands(t *testing.T) {
 	}
 }
 
-func TestRun_InsertDefaultCommand(t *testing.T) {
+func Test_insertDefaultCommand(t *testing.T) {
```

Comment:
Why not `Test_run_InsertDefaultCommand` here?

Reply:
because this [is testing the function `insertDefaultCommand`, not `run`](https://github.com/google/osv-scanner/blob/main/cmd/osv-scanner/main_test.go#L1006) (this was actually the original test I noticed, which lead me to realizing about the others 😅)